### ISSUE TITLE
Finalize README for v1.0

### ISSUE THREAD

**Issue by mickmister** [mattermost/mattermost-plugin-mscalendar#51]

**Comment by larkox** [comment]
I understand this issue is related with #6 , right?

**Comment by mickmister** [comment]
@larkox Yes I'd say they are the same. I'll close the other one since the README exists and is (fairly) mature. One major thing that has changed is that we don't have the dedicated bot user anymore.

**Comment by levb** [comment]
@aaronrothschild @justinegeffen should this issue be closed now?
